ci_framework/roles/devscripts/README.md

```diff
@@ -12,14 +12,17 @@ network configuration, repository setup and libvirt networks.
 ## Parameters
 
 * `cifmw_devscripts_artifacts_dir` (str) path to the directory to store the role artifacts.
-* `cifmw_devscripts_ci_token` (str) oAuth token required for accessing console.redhat.com.
-* `cifmw_devscripts_ci_token_file` (str) oAuth token file required for accessing console.redhat.com.
-   We can use `cifmw_devscripts_ci_token` or `cifmw_devscripts_ci_token_file` for passing OAuth token.
+* `cifmw_devscripts_ci_token` (str) oAuth token required for accessing
+  [openshift-console](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/).
```

Comment:
is that URI public?

Reply:
I believe so... I remember accessing it without VPN

Reply:
It is a public URI